packages/core/src/app/common/utility/isFloatingLabelEnabled.ts

```diff
@@ -0,0 +1,9 @@
+import { CheckoutContextProps } from '../../checkout';
+
+export default function isFloatingLabelEnabled(context: CheckoutContextProps): boolean {
```

Comment:
Instead of passing `CheckoutContextProps`, which contains both `CheckoutService` and `CheckoutState`, I'd just pass the minimum that you need, which is i.e.: `settings: CheckoutSettings`

Reply:
Sure, using `isFloatingLabelEnabled(checkoutSettings: CheckoutSettings): boolean` now.